x-pack/test/functional/services/transform/edit_flyout.ts

```diff
@@ -35,6 +35,12 @@ export function TransformEditFlyoutProvider({ getService }: FtrProviderContext)
       );
     },
 
+    // for now we expect this to be used only for opening the accordion
+    async clickTransformEditAccordionAdvancedSettings() {
+      await testSubjects.click('transformEditAccordionAdvancedSettings');
+      await testSubjects.existOrFail('transformEditAccordionAdvancedSettingsContent');
```

Comment:
While this is working fine, I think we can improve this a bit for future re-usability.
I'd suggest to make the action name a bit more specific (a `click` here can open or close the advanced settings section). So maybe something like `openTransformEditAccordionAdvancedSettings` or `ensureTransformEditAccordionAdvancedSettingsOpen` with an implementation similar to what we have in [AD job wizards](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/ml/job_wizard_common.ts#L500).

Reply:
I updated the method name in ae54ba5. Since there are more PRs planned anyway for some code consolidation, I'll do any further improvements in one of those. I added a note to the meta issue here: https://github.com/elastic/kibana/issues/77604